src/qt/zc2sigmapage.h

```diff
@@ -29,21 +31,26 @@ QT_END_NAMESPACE
 class Zc2SigmaPage : public QWidget
 {
     Q_OBJECT
-
 public:
-
     explicit Zc2SigmaPage(const PlatformStyle *platformStyle, QWidget *parent);
     ~Zc2SigmaPage();
-
     void createModel();
 
+    static bool showZc2SigmaPage();
+
 private:
     Ui::Zc2SigmaPage *ui;
     std::shared_ptr<Zc2SigmaModel> model;
+    std::shared_ptr<QTimer> tmrAvailMints;
```

Comment:
Same as previous issue. We can subscribe to the same event that I commented before.

Reply:
Not this time as the user can remint his ZC and we'd need to exclude this from the list before the next block comes